articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+   ![Networking option in New blade](images/azs-browser-new-networking.png)
+
+3. In the **Featured** section, select **Local network gateway**.
+
+   ![Create new networking resource](images/azs-browser-networking-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   - **Name** - The name of the local network gateway.
+
+   - **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address, as this will need to be updated once you have generated the public ip with the connection.
+
+   - **Address space** - One or more IP address ranges (in CIDR notation) that define your local network's address space. For example: 192.168.0.0/16. If you plan to use this local network gateway in a BGP-enabled connection, then the minimum prefix you need to declare is the host address of your BGP Peer IP address on your VPN device.
+
+   - **Configure BGP settings (Optional)** - Used only for when configuring BGP.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+     ![Create new local network gateway](images/azs-browser-create-Lnetwork-gateway.png)
+
+## Create the VPN connection in Azure Stack
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your public Azure VPN:
+
+1. Navigate to your virtual network gateway by clicking **All services** in the favourites panel, then selecting **Virtual network gateways** under the networking section.
+
+2. Select your virtual network gateway from the list.
+
+3. In the blade for your virtual network gateway, under the **Settings** section, select **Connections**.
+
+4. Click the **Add** button.
+
+5. In the **Add connection** blade, enter the following information:
+
+   - **Name** - The name of the connection.
+
+   - **Connection type** - Select **Site-to-site (IPSec)**.
+
+   - **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   - **Local network gateway**- Select the local network gateway you created earlier.
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - The resource group cannot be changed if you are adding a connection for an existing peer.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+     ![Add new connection](images/azs-browser-add-connection.png)
+
+6. Click **OK**.
+
```

Comment:
Add the extra level of headings as before for Azure Stack

## Set up VPN on the Azure Stack side

Then the following sections that provide steps for Public Azure use the next level of headings (###)